### PR TITLE
kubernete: no need to generate sha256 files for licenses

### PR DESCRIPTION
**Pull request by jaxesn**
During release we generate sha256 for all the files in the asset folder and upload to s3.  We have been doing this for everything in the asset folder, including the licenses files which feels unnecessary.  This tightens up the find a bit to just the tarballs and the binaries.

/hold

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by abhay-krishna** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/aws/eks-distro/pull/586#" title="Author self-approved">jaxesn</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=aws%2Feks-distro).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/aws/eks-distro/blob/main/OWNERS)~~ [jaxesn]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by TerryHowe — APPROVED** [review]
/lgtm

**Comment by abhay-krishna** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/aws/eks-distro/pull/586#" title="Author self-approved">jaxesn</a>*, *<a href="https://github.com/aws/eks-distro/pull/586#pullrequestreview-767987125" title="LGTM">TerryHowe</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=aws%2Feks-distro).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/aws/eks-distro/blob/main/OWNERS)~~ [TerryHowe,jaxesn]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by abhay-krishna** [comment]
New changes are detected. LGTM label has been removed.

**Comment by abhay-krishna** [comment]
@jaxesn: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
kubernetes-1-19-presubmit | a550c83a2abfe61f976e85faa2f5a09b83148b74 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro/586/kubernetes-1-19-presubmit/1443959582564028416) | `/test kubernetes-1-19-presubmit`

[Full PR test history](https://prow.eks.amazonaws.com/pr-history?org=aws&repo=eks-distro&pr=586). [Your PR dashboard](https://prow.eks.amazonaws.com/pr?query=is:pr+state:open+author:jaxesn).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by abhay-krishna** [comment]
@jaxesn: The following tests **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
kubernetes-1-19-presubmit | a550c83a2abfe61f976e85faa2f5a09b83148b74 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro/586/kubernetes-1-19-presubmit/1443959582564028416) | `/test kubernetes-1-19-presubmit`
kubernetes-1-18-presubmit | a550c83a2abfe61f976e85faa2f5a09b83148b74 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro/586/kubernetes-1-18-presubmit/1443959582517891072) | `/test kubernetes-1-18-presubmit`

[Full PR test history](https://prow.eks.amazonaws.com/pr-history?org=aws&repo=eks-distro&pr=586). [Your PR dashboard](https://prow.eks.amazonaws.com/pr?query=is:pr+state:open+author:jaxesn).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by abhay-krishna** [comment]
@jaxesn: The following tests **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
kubernetes-1-19-presubmit | a550c83a2abfe61f976e85faa2f5a09b83148b74 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro/586/kubernetes-1-19-presubmit/1443959582564028416) | `/test kubernetes-1-19-presubmit`
kubernetes-1-18-presubmit | a550c83a2abfe61f976e85faa2f5a09b83148b74 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro/586/kubernetes-1-18-presubmit/1443959582517891072) | `/test kubernetes-1-18-presubmit`
kubernetes-1-20-presubmit | a550c83a2abfe61f976e85faa2f5a09b83148b74 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro/586/kubernetes-1-20-presubmit/1443959582610165760) | `/test kubernetes-1-20-presubmit`

[Full PR test history](https://prow.eks.amazonaws.com/pr-history?org=aws&repo=eks-distro&pr=586). [Your PR dashboard](https://prow.eks.amazonaws.com/pr?query=is:pr+state:open+author:jaxesn).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by abhay-krishna** [comment]
@jaxesn: The following tests **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
kubernetes-1-19-presubmit | a550c83a2abfe61f976e85faa2f5a09b83148b74 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro/586/kubernetes-1-19-presubmit/1443959582564028416) | `/test kubernetes-1-19-presubmit`
kubernetes-1-18-presubmit | a550c83a2abfe61f976e85faa2f5a09b83148b74 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro/586/kubernetes-1-18-presubmit/1443959582517891072) | `/test kubernetes-1-18-presubmit`
kubernetes-1-20-presubmit | a550c83a2abfe61f976e85faa2f5a09b83148b74 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro/586/kubernetes-1-20-presubmit/1443959582610165760) | `/test kubernetes-1-20-presubmit`
kubernetes-1-21-presubmit | a550c83a2abfe61f976e85faa2f5a09b83148b74 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro/586/kubernetes-1-21-presubmit/1443959582652108800) | `/test kubernetes-1-21-presubmit`

[Full PR test history](https://prow.eks.amazonaws.com/pr-history?org=aws&repo=eks-distro&pr=586). [Your PR dashboard](https://prow.eks.amazonaws.com/pr?query=is:pr+state:open+author:jaxesn).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Review by TerryHowe — APPROVED** [review]
/lgtm

**Comment by abhay-krishna** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/aws/eks-distro/pull/586#" title="Author self-approved">jaxesn</a>*, *<a href="https://github.com/aws/eks-distro/pull/586#pullrequestreview-769278173" title="LGTM">TerryHowe</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=aws%2Feks-distro).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/aws/eks-distro/blob/main/OWNERS)~~ [TerryHowe,jaxesn]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by abhay-krishna** [comment]
New changes are detected. LGTM label has been removed.

**Review by kschumy — APPROVED** [review]
/lgtm

**Comment by abhay-krishna** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/aws/eks-distro/pull/586#" title="Author self-approved">jaxesn</a>*, *<a href="https://github.com/aws/eks-distro/pull/586#pullrequestreview-769338470" title="LGTM">kschumy</a>*, *<a href="https://github.com/aws/eks-distro/pull/586#pullrequestreview-769278173" title="LGTM">TerryHowe</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=aws%2Feks-distro).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/aws/eks-distro/blob/main/OWNERS)~~ [TerryHowe,jaxesn,kschumy]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by abhay-krishna** [comment]
New changes are detected. LGTM label has been removed.

**Pull request merged**
